README.md

```diff
@@ -14,7 +14,7 @@
 
 ## Website
 
-Soon.
+Click [this link](https://mullenlucas.github.io/prog-rock-concert-sched) to access the resulting website
```

Comment:
- [ ] As written in the project requirements, I kindly suggest that you add a video demo of your project presenting the features of your project. The video should consist of the following:

- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.